src/component/material-ui/nav-wrapper/index.js

```diff
@@ -87,25 +128,25 @@ class NavWrapper extends Component {
 
     const loginSignupJSX = (
       <RaisedButton
-        label="Signup/Login"
+        className="login-logout-button"      
         onClick={(event) => this.setState({
           anchorEl: event.target,
           popoverOpen: true,
         })}
-      />
+      >
+        Signup / Login
+      </RaisedButton>
     );
 
     const authButton = this.props.loggedIn ? logoutJSX : loginSignupJSX;
-
+    
+    // Seth - Using location.pathname to conditional render savedButton with switch function
+    const savedButton = this.props.loggedIn ?
+      this.renderSavedSwitch(this.props.location.pathname) : null;
+    
     return (
-      <Paper
```

Comment:
Why are you removing the paper? Isn't that what gives it the theming to keep it consistent with our current setup?

Reply:
the paper was put in for the nave bar, but then the background and shadow were set to transparent so it was acting just live a normal div.

Reply:
The Paper is essentially just used to get that "paper" look